lib/Remote/RemoteWebDriver.php

```diff
@@ -141,9 +141,16 @@ public static function create(
      * @param string $session_id The existing session id
      * @return RemoteWebDriver
      */
-    public static function createBySessionID($session_id, $selenium_server_url = 'http://localhost:4444/wd/hub')
+    public static function createBySessionID($session_id, $selenium_server_url = 'http://localhost:4444/wd/hub', $connection_timeout_in_ms = null, $request_timeout_in_ms = null)
```

Comment:
Could you please update the phpDoc above to reflect the new params? It will be good to merge for me then!

Reply:
Done